### ISSUE TITLE
Wayland improvements

### ISSUE THREAD

**Issue by ninja-** [flathub/org.prismlauncher.PrismLauncher#74]
FYI, a couple things that could be worked upon to complete Wayland experience:
- Implement [xdg_toplevel_icon_manager_v1](https://wayland.app/protocols/xdg-toplevel-icon-v1#xdg_toplevel_icon_manager_v1) to set the Minecraft icon instead of Wayland icon. For now there's no compositors supporting it, earliest will be KDE 6.3 so that will take a while.
- Allow overriding xdg app id via env in GLFW; this could be used for launcher-less GLFW games or in case of no xdg_toplevel_icon_manager_v1 linking the game window to the launcher as it's better than nothing
- F11 always moves the window to primary display instead of the display where the window was; this works correctly on Xwayland...
- Cursor theme doesn't match system and it's sizing is invalid because scale is not multiplied by fractional-scale-v1; libdecor has code that pulls these from xdg-portal but idk if glfw even uses this code or has it's own 
- hitting escape doesn't center the cursor; this can be done using https://wayland.app/protocols/pointer-constraints-unstable-v1#zwp_confined_pointer_v1 I think; it's not very popular but for example KiCad was playing with this

**Comment by orowith2os** [comment]
Yes, these problems are not specific to the Flatpak. Please file these issues with the appropriate projects, they don't sound like PrismLauncher issues.

**Comment by Scrumplex** [comment]
Closing this as this is not a packaging issue. See https://github.com/PrismLauncher/PrismLauncher/issues/new/choose
